js/actions/bookmarkActions.js

```diff
@@ -4,15 +4,18 @@
 
 'use strict'
 
+const Immutable = require('immutable')
 const siteUtil = require('../state/siteUtil')
 const appActions = require('./appActions')
 const windowActions = require('./windowActions')
 const eventUtil = require('../lib/eventUtil')
+const appStoreRenderer = require('../stores/appStoreRenderer')
 const {SWITCH_TO_NEW_TABS} = require('../constants/settings')
 const getSetting = require('../settings').getSetting
 
 const bookmarkActions = {
-  openBookmarksInFolder: function (allBookmarkItems, folderDetail) {
+  openBookmarksInFolder: function (folderDetail) {
```

Comment:
For example this refactoring
https://github.com/brave/browser-laptop/pull/9678/files#diff-de5d4f1b019deecc5b1926970450546cR392

https://github.com/brave/browser-laptop/pull/9678/files#diff-733d30fbf34816322d3f25dd59c1eb99R747

I know that is in anti-pattern, but I am not sure how to achieve it otherwise (I already talked with @bsclifton about it), so I am opened for any suggestions how to fix this and have only one action per call. I didn't find any good approach with the current implementation.